content/en/docs/hertz/tutorials/toolkit/toolkit.md

```diff
@@ -0,0 +1,1019 @@
+
+Now let's develop the "Method2" interface
+
+```go
+// Method1 .
+// @router /hello [GET]
+func Method1(ctx context.Context, c *app.RequestContext) {
+   var err error
+   var req hello.HelloReq
+   err = c.BindAndValidate(&req)
+   if err != nil {
+      c.String(400, err.Error())
+      return
+   }
+
+   resp := new(hello.HelloResp)
+
+   // You can modify the logic of the entire function, not just the current template
+   resp.RespBody = "hello," + req.Name // added logic
+
+   c.JSON(200, resp)
+}
+
+// Method2 .
+// @router /other [POST]
+func Method2(ctx context.Context, c *app.RequestContext) {
+   var err error
+   var req hello.OtherReq
+   err = c.BindAndValidate(&req)
+   if err != nil {
+      c.String(400, err.Error())
+      return
+   }
+
+   resp := new(hello.OtherResp)
+
+   // added logic
+   resp.Resp = "Other method: " + req.Other
+
+   c.JSON(200, resp)
+}
+```
+
+4. Compile the project
+
+```bash
+go build
+```
+
+5. Run the project and test it
+
+Run the project:
+
+```bash
+./{{your binary}}
+```
+
+Test：
+
+```bash
+curl --location --request POST 'http://127.0.0.1:8888/other' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "Other": "other method"
+}'
+```
+
+If it returns `{"Resp":"Other method: other method"}`, it means that the interface is connected.
+
+### The structure of the generated code
+
+The structure of the code generated by hz is similar. The following is an example of the structure of the code generated by the section "Create a project based on thrift IDL" to illustrate the meaning of the code generated by hz.
+
+```bash
+.
+├── biz                                // business layer, which stores business logic related processes
+│   ├── handler                        // store handler file
+│   │   ├── hello                      // hello/example corresponds to the namespace defined in thrift IDL; for protobuf IDL, it corresponds to the last level of go_package
+│   │   │   └── example
+│   │   │       ├── hello_service.go   // the handler file, the user implements the method defined by the IDL service in this file, and will search for the existing handler in the current file when updating, and append a new handler to the end
+│   │   │       └── new_service.go     // same as above, each service defined in IDL corresponds to a file
+│   │   └── ping.go                    // ping handler carried by default, used to generate code for quick debugging, no other special meaning
+│   ├── model                          // IDL content-related generation code
+│   │   └── hello                      // hello/example corresponds to the namespace defined in thrift IDL; for protobuf IDL, it corresponds to the last level of go_package
+│   │     └── example
+│   │         └── hello.go             // the product of thriftgo, the go code containing the contents of hello.thrift definition, will be regenerated on update
```

Comment:
the product of thriftgo. It contains go code generated from the contents of the hello.thrift definition. And it will be regenerated when use "update" command.

Reply:
fixed.